ambry-router/src/main/java/com.github.ambry.router/PutOperation.java

```diff
@@ -891,6 +907,7 @@ private void encryptChunk() {
     void onFillComplete(boolean updateMetric) {
       buf.flip();
       chunkBlobSize = buf.remaining();
+      chunkFillCompleteAtMs = time.milliseconds();
```

Comment:
where is this used?

Reply:
check line 318 within onChunkOperationComplete()